interface/src/app/tribe_client/genesets.js

```diff
@@ -2,38 +2,7 @@ angular.module('adage.tribe_client.genesets', [
   'adage.tribe_client.resource'
 ])
 
-// Geneset Search Factory
-// Allows for retreiving for and interacting with search results for genesets.
-.factory('GenesetSearch', ['Genesets', function(Genesets) {
-  var genesets = [];
-  var query = {};
-  var resultCount = null;
-
-  return {
-    getQuery: function() {
-      return query;
-    },
-    getGenesets: function() {
-      return genesets;
-    },
-    clear: function() {
-      query = {};
-      genesets = [];
-    },
-    getResultCount: function() {
-      return resultCount;
-    },
-    query: function(searchParams) {
-      query = searchParams;
-      Genesets.query(query, function(data) {
-        resultCount = data.meta.total_count;
-        genesets = data.objects;
-      });
-    }
-  };
-}])
-
-.directive('genesetSearchForm', ['GenesetSearch', function(GenesetSearch) {
```

Comment:
Good call on just taking out the `GenesetSearch` service entirely. I guess it could have been a clue for me that I was pointing out all the places you were no longer using it... :-)